shell/browser/api/electron_api_session.cc

```diff
@@ -571,6 +571,7 @@ v8::Local<v8::Promise> Session::ClearAuthCache() {
       ->GetNetworkContext()
       ->ClearHttpAuthCache(
           base::Time(),
```

Comment:
should this be `base::Time::Min()`? The linked CL seems to use it, and they are subtly different (`base::Time()` is t=0, `base::Time::Min()` is t=`std::numeric_limits<int64_t>::min()`, and the `ClearHttpAuthCache` code specifically checks for `base::Time::Min()`)
```suggestion
          base::Time::Min(),
```

Reply:
I feel like in all cases these are equiv and we use `base::Time()` all over this file.  I can change it but I'm pretty sure it's the same.

Reply:
It's not, in this specific case.

https://source.chromium.org/chromium/chromium/src/+/master:net/http/http_auth_cache.cc;l=305;drc=aeda2e228a3312b75ba84c92821528251958c739

`base::Time().is_min() == false`.

Reply:
Sorry I meant functionally, like it would result in the same things being cleared.  I think it's a perf optimization though so I'll follow this up with a PR to change all our `base::Time()` usage in session to use `Min()`

Reply:
yeah, i'm not sure in what cases you could end up with auth cache entries with negative times. Perhaps if you're loading up a profile you last logged in from in 1963?